### DISCUSSION TITLE
Is there a way to automate "setup"?

### DISCUSSION THREAD

**Discussion by ninthy** [dyc3/steamguard-cli#421] [Q&A]
I tried but it doesnt work with flags, any suggestions?

**Comment by dyc3** [comment]
No. The setup process is heavily dependent on what is already set on the account, and grabbing codes from email or sms.

What's your use case?

**Reply by hrxn** [reply]
> What's your use case?

I suspect something nefarious. Hands down, I get why an individual person would run a couple (like, a max of five or something) steam accounts, I think there are legitimate reasons for that, but automation of account setup etc. implies a lot more than just a handful of accounts.

I can't think of a use case here that's legitimate.
